addon/PlacesProvider.js

```diff
@@ -321,6 +322,12 @@ Links.prototype = {
     }
   },
 
+  getTopNewTabSites() {
```

Comment:
Fair enough :). How about let's call it "asyncGetTopNewTabSites" like other functions in this module?